pkg/providers/vsphere/vsphere_test.go

```diff
@@ -1309,7 +1337,7 @@ func TestSetupAndValidateCreateClusterNoServer(t *testing.T) {
 
 	err := provider.SetupAndValidateCreateCluster(ctx, clusterSpec)
 
-	thenErrorExpected(t, "failed setup and validations: VSphereDatacenterConfig server is not set or is empty", err)
+	thenErrorExpected(t, "VSphereDatacenterConfig server is not set or is empty", err)
```

Comment:
The only change here is the error is not wrapped anymore bc now it's part of the validator and not the env var setup